ApplicationConfig/Providers/SiteAccesses.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\ApplicationConfig\Providers;
+
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use EzSystems\PlatformUIBundle\ApplicationConfig\Provider;
+
+class SiteAccesses implements Provider
```

Comment:
> makes sense to use first one if no languageCode is set on creation

that's exactly the point of the issue (reminder: unlike legacy, when creating a Content item in PlatformUI, you don't choose a language so the content creation is always initiated in a "default language").

For the Content Type, that's right but that's a different issue (see https://github.com/ezsystems/PlatformUIBundle/pull/581#discussion_r62660891).